PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java

```diff
@@ -54,7 +56,10 @@ public void setSubscription(Subscription subscription) {
         if (Objects.isNull(subscription)) {
             return new ArrayList<>();
         }
-        List<? extends Region> regions = Azure.az(AzureAccount.class).listRegions(subscription.getId());
+        if (loader == null) {
```

Comment:
May merge with above condition

Reply:
Two regioncombobox is ugly, will remove it later.